mopidy/file/library.py

```diff
@@ -142,4 +142,7 @@ def _is_in_basedir(self, local_path):
             for media_dir in self._media_dirs)
 
     def _get_path_for_display(self, path):
-        return bytes(path).decode('utf-8', 'replace')
```

Comment:
Is this not always a `Path` instance now?

Reply:
I knew this was familiar! I totally forgot you address it here, just differently. Sorry!

I guess it was just confusing because `path` isn't a path.

Reply:
I'm so looking forward to type checking and pruning isinstance checks!